compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -549,7 +549,8 @@ loco::NodeShape infer_concatenation(const luci::CircleConcatenation *node)
       if (j == static_cast<uint32_t>(axis))
         output_shape.dim(j) = output_shape.dim(j).value() + input_shape.dim(j).value();
```

Comment:
Then for this line, if we add conditions, would it be like this?
```cxx
assert(output_shape.dim(j).known() && input_shape.dim(j).known());
output_shape.dim(j) = output_shape.dim(j).value() + input_shape.dim(j).value();
```
if so, how about adding it?

Reply:
It is because, using `known()` is added existing codes which we didn't check but for now, it would be better to check..

Reply:
I will add a comment about this :)